### PR TITLE
Add SANS

### PR DESCRIPTION
**Pull request by rwehe**
adding the SANS Institute

### CODE REVIEW

**Comment by rwehe** [comment]
Hmm, not sure why this is failing workflow. It appears that it has something to do with the Facebook link? 
https://www.facebook.com/sansinstitute is valid for me.

**Comment by hkamran80** [comment]
The Facebook test has been having problems because of Facebook's API

**Comment by rwehe** [comment]
> The Facebook test has been having problems because of Facebook's API

Understood, let me know if anything else is required for this PR. Thanks!

File: img/s/sans.org.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...6 3.614 24 4.949 24c2.288 0 3.909-2.27 3.909-5.956 0-2.962-1.367-5.452-3.4-8.508M43.074 23.37c-1.844 0-3.517-2.284-4.022-6.309h4.721c.84 1.51 1.304 2.853 1.304 3.98 0 1.418-.858 2.33-2.003 2.33zm-16.926-7.063V4.306l5.24 12h-5.24zm-.888 3.342c0 2.236-.01 3.323-1.6 3.323h-1.006l-1.036-5.91h3.642v2.587zm-11.237-3.342 1.989-11.672 1.925 11.672h-3.914zM43.71 9.536c-1.874-2.867-3.59-4.663-3.59-6.648 0-1.449.73-2.34 1.874-2.34 1.685 0 3.083 2.554 3.464 7.081h.478V.335h-.478l-.699 1.387C43.997.682 42.98.02 41.994.02c-1.938 0-3.495 1.906-3.495 5.277 0 3.56 1.557 6.032 3.4 8.805.513.767.972 1.575 1.37 2.204h-4.306c-.023-.503-.04-.629-.051-1.258h-.713v1.258h-3.045V3.676c0-1.985.127-2.71 1.776-2.678V.335h-4.947v.663c1.522 0 2.157.914 2.157 2.993V14.42L28.49.335h-4.626v.63h1.395v15.342h-3.797L18.68.335h-2.797L12.01 19.727c-.382 1.953-.321 2.946-2.224 3.23v.644h5.2v-.644c-1.394-.032-1.774-.41-1.774-1.418 0-.441.034-.821.065-1.104l.575-3.374h4.258l.954 5.911H17.27v.629h11.543v-.644c-1.776-.126-2.664-1.135-2.664-3.497v-2.4h5.607l2.742 6.54h.657v-6.54H38.2v6.791h.713l.858-1.676C40.85 23.436 41.867 24 43.202 24c2.288 0 3.909-2.27 3.909-5.956 0-2.962-1.367-5.452-3.4-8.508z" mask="url(#b)"/></g></svg>
```
Comment (hkamran80) [inline img/s/sans.org.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 47.1 23.98"><defs><path id="a" d="M.01.02h47.1V24H.01Z"/></defs><g fill="none" transform="translate(-.01 -.02)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path fill="#004f6e" d="M5.46 9.54c-1.88-2.87-3.6-4.67-3.6-6.65 0-1.45.74-2.34 1.88-2.34C5.43.55 6.82 3.1 7.2 7.63h.54V.33H7.2l-.7 1.4C5.75.67 4.74.01 3.75.01 1.8.02.24 1.92.24 5.3c0 3.56 1.56 5.96 3.4 8.73 2 3 3.18 5.22 3.18 7.02 0 1.41-.85 2.21-2 2.21-2.1 0-3.97-2.8-4.16-8.21H0v8.8h.66l.86-1.67C2.6 23.44 3.62 24 4.95 24c2.29 0 3.9-2.27 3.9-5.96 0-2.96-1.36-5.45-3.4-8.5m37.62 13.83c-1.84 0-3.51-2.28-4.02-6.3h4.72a8.54 8.54 0 0 1 1.3 3.97c0 1.42-.85 2.33-2 2.33zm-16.92-7.06v-12l5.24 12h-5.24zm-.89 3.34c0 2.23-.01 3.32-1.6 3.32h-1l-1.04-5.9h3.64ZM14.02 16.3l2-11.67 1.92 11.67Zm29.69-6.77c-1.87-2.87-3.59-4.67-3.59-6.65 0-1.45.73-2.34 1.87-2.34 1.69 0 3.09 2.55 3.47 7.08h.48V.33h-.48l-.7 1.4C44 .67 42.98.01 41.99.01c-1.93 0-3.5 1.9-3.5 5.28 0 3.56 1.57 6.03 3.4 8.8.52.77.98 1.58 1.38 2.2h-4.3c-.03-.5-.05-.62-.06-1.25h-.71v1.26h-3.05V3.68c0-1.99.13-2.71 1.78-2.68V.33h-4.95V1c1.53 0 2.16.91 2.16 3v10.42L28.49.34h-4.63v.63h1.4V16.3h-3.8L18.68.34h-2.8l-3.87 19.39c-.38 1.95-.32 2.94-2.22 3.23v.64h5.2v-.64c-1.4-.04-1.78-.41-1.78-1.42 0-.44.04-.82.07-1.1l.57-3.38h4.26l.95 5.91h-1.79v.63h11.54v-.64c-1.77-.13-2.66-1.14-2.66-3.5v-2.4h5.6l2.75 6.54h.65v-6.54h3.05v6.8h.71l.86-1.69c1.08 1.27 2.1 1.83 3.43 1.83 2.29 0 3.91-2.27 3.91-5.96 0-2.96-1.37-5.45-3.4-8.5z" mask="url(#b)"/></g></svg>
```

File: entries/s/sans.org.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "SANS": {
+    "domain": "sans.org",
+    "contact": {
+      "facebook": "sansinstitute",
```
Comment (hkamran80) [inline entries/s/sans.org.json]:
```suggestion
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/s/sans.org.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "SANS": {
+    "domain": "sans.org",
+    "contact": {
+      "twitter": "SANSInstitute"
```
Comment (hkamran80) [inline entries/s/sans.org.json]:
```suggestion
      "twitter": "SANSInstitute",
      "email": "support@sans.org"
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**